tests/unit/preprocessor/_area/test_area.py

```diff
@@ -623,9 +623,11 @@ def test_extract_shape(make_testcube, square_shape, tmp_path, crop, ids):
 def test_extract_shape_natural_earth(make_testcube, ne_ocean_shapefile):
     """Test for extracting a shape from NE file."""
     expected = np.ones((5, 5))
+    preproc_path = Path(esmvalcore.preprocessor.__file__).parent
+    shp_file = preproc_path / "ne_masks" / "ne_50m_ocean.shp"
     result = extract_shape(
         make_testcube,
-        "esmvalcore/preprocessor/ne_masks/ne_50m_ocean.shp",
+        shp_file,
```

Comment:
Does this work? The `extract_shape` docstring says it needs a string as the function argument.

Reply:
I see that `fiona.open` will take a `Path` object 👍

Reply:
yes it does, test passes both run locally and from within the conda build environment which is done outside the dir